prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -73,6 +73,20 @@ presubmits:
       nodeSelector:
         testing: build-pool
 
+  - name: integ-istioctl-local-presubmit-tests-master
+    <<: *job_template
+    context: prow/integ-istioctl-local-presubmit-tests.sh
+    always_run: true
+    optional: true
+    spec:
+      containers:
+        - <<: *istio_container
+          command:
+            - entrypoint
+            - prow/integ-istioctl-local-presubmit-tests.sh
+      nodeSelector:
+        testing: build-pool
```

Comment:
any reason why you need to use build-pool for this test and not use test-pool?

Reply:
Just following the pattern in the same file. Unit tests are also ran in ta similar way.

Reply:
let's change the tests to use "test-pool" instead of "build-pool". We want to use "build-pool" just for building images if possible..

Reply:
I changed the new tests to use test-pool. We can change the existing ones to use test-pool separately.

Reply:
I agree.